### DISCUSSION TITLE
Extra art possibility in Kodi?

### DISCUSSION THREAD

**Discussion by marek256** [croneter/PlexKodiConnect#1294] [Q&A] [answered]
Hello
I apologize in advance for bothering, but I am not able to find proper answer on google.
I used to use your addon for Kodi, but recently I had to end up using NAS so no more PlexKodiConnect.
At the moment I "stream" movies from huge external hdd only and what I miss in Kodi now is the extra fanart which I can see when I play video. Kodi does not have this function and I just would like to know if it is connected to Plex only or if it is simply some Kodi addon I can use.
I attached you a simple screenshot in a link below to show you simply what I mean.

Any answer will be appreciated.

Thank you in advance

[https://ibb.co/DpJ27m6]

**Comment by croneter** [comment, marked as answer]
You can use the Kodi add-on "Artwork beef", for example

**Comment by Riko-sketch** [comment]
ive never seen kodi be for i may try it out
